_episodes/07-branches.md

```diff
@@ -6,8 +6,10 @@ questions:
 - "What are branches?"
 - "How can I work in parallel using branches?"
 objectives:
-- "Understand why branches are useful for working on separate tasks in the same repository."
-- "Use branches for working in parallel"
+- "Understand why branches are useful for:"
+- "working on separate tasks in the same repository concurrently"
```

Comment:
Ah yes! I tried doing exactly that but jekyll was not a fan for some reason. I meant to revisit this issue before creating the PR. Thank you reminding me! I'll work on this straightaway.

- [ ] Nest bullets on why branches are useful